src/components/FormGroup.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 
 const FormGroup = (props) => {
-  const { label, ...rest } = props
+  console.log(props);
+  const { label, ...rest } = props;
   const { styles } = FormGroup;
 
+  function inputField() {
+    if (props.placeholder === 'what are you looking for?') {
```

Comment:
I tried adding a prop blackText={true} to FormGroup on line 103 in Dashboard.jsx, but was getting an error in the console...